RoundTimer.py

```diff
@@ -42,21 +46,22 @@ def kill(signal, frame):
 # Tread creation
 logging.info("Creating threads")
 clkThread = Clock(clock, (64, 32), position=(0, 0), color=(255, 0, 0, 255))
-threads.append(clkThread)
 
-tmrThread1 = Timer(timers, (32, 16), minutes=1, position=(0, 0), offset=(0, -6), color=(0, 0, 255, 255))
-tmrThread2 = Timer(timers, (32, 16), minutes=1, position=(0, 16), offset=(0, -6), color=(255, 255, 0, 255))
-tmrThread3 = Timer(timers, (32, 16), minutes=2, position=(32, 0), offset=(0, -6), color=(255, 0, 0, 255))
-tmrThread4 = Timer(timers, (32, 16), minutes=2, position=(32, 16), offset=(0, -6), color=(255, 255, 255, 255))
+tmrThreads = []
```

Comment:
At some point, we will want to move this to some sort of config file. For now, we can leave it as hard-coded but it needs to be more configurable going forward. Maybe open an issue once this merges?